provider/postgis/README.md

```diff
@@ -6,24 +6,60 @@ The PostGIS provider manages querying for tile requests against a Postgres datab
 [[providers]]
 name = "test_postgis"       # provider name is referenced from map layers (required)
 type = "postgis"            # the type of data provider must be "postgis" for this data provider (required)
-host = "localhost"          # PostGIS database host (required)
-port = 5432                 # PostGIS database port (required)
-database = "tegola"         # PostGIS database name (required)
-user = "tegola"             # PostGIS database user (required)
-password = ""               # PostGIS database password (required)
+
+uri = "postgres://tegola:supersecret@localhost:5432/tegola?sslmode=prefer" # PostGIS connection string (required)
+
+host = "localhost"                # PostGIS database host (deprecated)
+port = 5432                       # PostGIS database port (deprecated)
+database = "tegola"               # PostGIS database name (deprecated)
+user = "tegola"                   # PostGIS database user (deprecated)
+password = "supersecret"          # PostGIS database password (deprecated)
+max_connections = 10              # PostGIS max connections (deprecated)
+max_connection_idle_time = "30m"  # PostGIS max connection idle time (deprecated)
+max_connection_lifetime = "1h"    # PostGIS max connection life time (deprecated)
 ```
 
 ### Connection Properties
 
+Establishing a connection via connection string (`uri`) will become the default connection method as of v0.17.0.
+Connecting via host/port/database is flagged for deprecation as of v0.16.0 but will be possible until v0.17.0 still.
```

Comment:
minor detail, but change to v0.15.0 and v0.16.0. We still haven't release v0.15.0 (will do so very soon, I'm just waiting for confirmation that we can migrate to go-embed entirely from one of our users that has a build pipeline on go 1.15)